truss/remote/baseten/service.py

```diff
@@ -47,6 +47,14 @@ def decode_content():
 
             return decode_content()
 
+        parsed_response = response.json()
+
+        if "model_output" not in parsed_response:
```

Comment:
might be more clear to check for existence of the `error` key if we know it will always come with that